tests/scale/test_load.py

```diff
@@ -197,25 +197,36 @@ def test_scaling_load(master_count,
 
 
 @pytest.mark.scalecleanup
-def test_cleanup_scale(mom) -> None:
-    """Blanket clean-up of jenkins instances on a DC/OS cluster.
+def test_cleanup_scale(mom, min_index, max_index,) -> None:
+    """
+     Args:
+        mom: Marathon on Marathon instance name
+        min_index: minimum index to begin jenkins suffixes at
+        max_index: maximum index to end jenkins suffixes at
 
-    1. Queries Marathon for all apps matching "jenkins" prefix
+    Blanket clean-up of jenkins instances on a DC/OS cluster.
+
+    1. Queries Marathon for all apps matching "jenkins" prefix, or deletes range
     2. Delete all jobs on running Jenkins instances
     3. Uninstall all found Jenkins installs
     """
-    r = sdk_marathon.filter_apps_by_id('jenkins', mom)
-    jenkins_apps = r.json()['apps']
-    jenkins_ids = [x['id'] for x in jenkins_apps]
-
     service_ids = list()
-    for service_id in jenkins_ids:
-        if service_id.startswith('/'):
-            service_id = service_id[1:]
-        # skip over '/jenkins' instance - not setup by tests
-        if service_id == 'jenkins':
-            continue
-        service_ids.append(service_id)
+
+    if min_index != -1 and max_index != -1:
```

Comment:
Correct me if I am wrong.

The ticket mentions two things : 
```
1. Killing off a list of specified jenkins instances, likely a list we can supply at command line.
2. Killing off a contiguous range of jenkins instances, we can reuse the --min and --max index parameters for this.
```
I see 2 is accomplished via the `if` logic. Where is `#2` accomplished ?

Reply:
making a quick fix for #2 so it can be used in scale test tmmrw. Will follow up with a PR for #1, (specific instances can currently be killed off by making min == max)

Reply:
in that case, i guess we can keep the ticket open, but that's up to you.